File: Python/004_Lista_if_else/Ex_006.py
"""
6) Faça um programa que leia o salário de um trabalhador e o valor da prestação de um empréstimo. Se a
prestação for maior que 20% de seu salário, mostre a mensagem: “emprestimo nao concedido”. Caso
contrário, mostre a mensagem: “emprestimo concedido”
"""

salario = float(input("Digite o salario: "))
parcela = float(input("Digite a parcela: "))

if parcela > 0.2*salario:
    print("Emprestimo não Concedido")
else: 
    print("Emprestimo  concedido")

File: XDES01 - Linguagem C/004_Lista_if_else/Ex_006.c
//6) Faça um programa que leia o salário de um trabalhador e o valor da prestação de um empréstimo. Se a
//prestação for maior que 20% de seu salário, mostre a mensagem: “emprestimo nao concedido”. Caso
//contrário, mostre a mensagem: “emprestimo concedido”.
#include <stdio.h>
int main ()
{
  float salario, prestacao;

  printf("Digite o valor do seu salario:\n");
  scanf("%f", &salario);

  printf("Digite o valor do seu prestacao:\n");
  scanf("%f", &prestacao);

  if( prestacao > 0.2* salario)   
  {
    printf("emprestimo nao concedido\n");
  }
  else
    printf("emprestimo concedido\n");
  
  return 0;
  
}